zero/gkyl_dg_calc_canonical_pb_vars_priv.h

```diff
@@ -149,4 +161,18 @@ choose_canonical_pb_alpha_surf_v_kern(int dir, int cdim, int poly_order)
     return ser_canonical_pb_alpha_surfvz_kernels[cdim-1].kernels[poly_order];
   else
     return NULL;
+}
+
+GKYL_CU_D
+static canonical_pb_pressure_t
+choose_canonical_pb_pressure_kern(int dir, int cdim, int poly_order)
```

Comment:
This kernel selection is wrong. There is only one kernel selected so you should not be passing dir, and you're just calling the same return method three times. We should fix this. You only need cdim to select the correct pressure kernel.

Reply:
Fixed in commit https://github.com/ammarhakim/gkylzero/pull/364/commits/cd5dd0f23cdbc04e19cc228ce3e8d532a945dddc